docs/manipulation.rst

```diff
@@ -246,6 +246,43 @@ Or similarly, expressed in pixels:
     >>> spec_w_unc.uncertainty # doctest: +ELLIPSIS
     StdDevUncertainty([0.18714535, ..., 0.18714535])
 
+S/N Threshold Mask
+------------------
+
+It is useful to be able to find all the spaxels in an ND spectrum
```

Comment:
I simplified this a bit because most of the rest of this page's text is about more traditional 1d spectra.
```suggestion
There are times when it is useful to be able to find all the pixels in a spectrum
```